src/Razor/src/Microsoft.VisualStudio.LegacyEditor.Razor/Parsing/VisualStudioRazorParser.cs

```diff
@@ -341,6 +341,11 @@ private void TextBuffer_OnChanged(object sender, TextContentChangedEventArgs arg
                 codeDocument.Items[item.Key] = item.Value;
             }
 
+            if (currentCodeDocument.TryGetTagHelperContext(out var tagHelperContext))
+            {
+                currentCodeDocument.SetTagHelperContext(tagHelperContext);
```

Comment:
This looks like a no-op - getting the context and setting it back to the same code document.

Reply:
Oop! Great catch -- thank you!